extensions/GitExtension/FileExplorerGitIntegration/Models/GitLocalRepository.cs

```diff
@@ -37,9 +37,10 @@ internal GitLocalRepository(string rootFolder, RepositoryCache? cache)
             // Rather than open the repo from scratch as validation, try to retrieve it from the cache
             OpenRepository();
         }
-        catch
+        catch (Exception ex)
         {
-            throw new ArgumentException("Invalid repository path");
+            _log.Error(ex, "Exception thrown in OpenRepository");

```

Comment:
Do we have telemetry we can log this to?

Reply:
No telemetry has been added for the git extension. I do hope and will try to get to it soon.